classes/Currency.php

```diff
@@ -379,6 +438,20 @@ public function getName()
         return Tools::ucfirst($this->name[$id_lang]);
     }
 
+    public function getSymbol()
+    {
+        if (is_string($this->symbol)) {
```

Comment:
What if `$this->symbol` is empty?

Reply:
It will return an empty string then but I think that's what we want as long as the value comes from the DB. Wdyt?

Reply:
Oh yes, I think I'm really tired today :sweat_smile: